Changes.md

```diff
@@ -5,6 +5,18 @@
 HEAD
 ---------
 
+- Support job-specific log levels. This will allow you to turn on debugging for
+  problematic workers. [fatkodima, #INSERT_NUM_HERE]
+```ruby
+class MyWorker
+  include Sidekiq::Worker
+  sidekiq_options log_level: :debug
```

Comment:
Should we even document this usage pattern or will most people use it dynamically via `set` on the console?